src/components/WikiTags/WikiTags.js

```diff
@@ -176,6 +178,36 @@ const combineStyles = (...styles) => {
   };
 };
 
+async function getSessionReport(props, contextId) {
```

Comment:
For consistency purposes, this could be in api.js.

Reply:
You asked me to move that to the WikiTags.js component, in our meeting last month. I'll move it back, no problem.